authority/admin/api/admin.go

```diff
@@ -0,0 +1,139 @@
+package api
+
+import (
+	"net/http"
+
+	"github.com/go-chi/chi"
+	"github.com/smallstep/certificates/api"
+	"github.com/smallstep/certificates/authority/admin"
+	"go.step.sm/linkedca"
+)
+
+// CreateAdminRequest represents the body for a CreateAdmin request.
+type CreateAdminRequest struct {
+	Subject     string              `json:"subject"`
+	Provisioner string              `json:"provisioner"`
+	Type        linkedca.Admin_Type `json:"type"`
+}
+
+// Validate validates a new-admin request body.
+func (car *CreateAdminRequest) Validate() error {
+	return nil
```

Comment:
We should check the properties are not empty and the type is a valid one.

Reply:
Agreed. I'm not checking the provisioner create inputs either. I'm not going to have time to test those validators, so didn't want to get too deep.

Reply:
I can do some simple property checking though.